pkg/apis/core/type_seedconfig.go

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2021 "SAP SE or an SAP affiliate company and Gardener contributors"
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package core
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// The SeedConfig is created to define the configuration for a Kubernetes cluster, that can host Landscaper Service deployments.
+type SeedConfig struct {
```

Comment:
not sure if we alraedy talked about this. But seed is something in the context of gardener.
I think we should not reuse that specific name and rather call it something like worker.

Reply:
I think you came up with name ;-)
I always struggle with good names. Any suggestions?
`TargetConfig` or `ServiceTargetConfig` ... ?

Reply:
I would vote for ServiceTargetConfig or ServiceHostConfig

Reply:
I have renamed it to `ServiceTargetConfig`